345 - spec pvp timings fix

sad-emu opened the pull request:
Trying to fix #345 

Only affects pvp

Before the change

https://github.com/2006-Scape/2006Scape/assets/42761884/da8abc3d-fd8f-49bf-8241-5a8eb658798e



After the change

https://github.com/2006-Scape/2006Scape/assets/42761884/1c6fcc57-79b1-409a-8622-a7f34f3882b7

Commit:
345 - spec pvp timings fix

Dark98 commented on 2006Scape Server/src/main/java/com/rs2/game/content/combat/CombatAssistant.java:
```diff
@@ -30,6 +30,8 @@
 import com.rs2.world.Boundary;
 import com.rs2.world.clip.PathFinder;
 
+import static com.rs2.game.content.StaticItemList.GRANITE_MAUL;
```
We should probably just import StaticItemList because we'll need it for other magic numbers in this file

ipkpjersi replied:
Swarm is now ambiguous btw, since there's a swarm item and swarm NPC, so swarm will need to be updated to mention StaticNpcList most likely.

Dark98 committed:
Update CombatAssistant.java

Import staticitemlist.* instead as we'll be using it for other stuff in this file

Dark98 committed:
Merge branch 'master' into master

Dark98 committed:
Update CombatAssistant.java
